app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (OutdatedVaccineError,

```

Comment:
The section of PEP 8 concerning imports recommends grouping them in the following order:

    Standard library imports.
    Related third-party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.

Reply:
Added blank line